scripts/feather.sh

```diff
@@ -1,9 +1,19 @@
 #!/bin/bash -e
 
-node bin/generate-icon Feather/style.css\
+fontcustom compile node_modules/feather-icons/dist/icons\
```

Comment:
Add something to let users know how to fix this if it fails here, just check the status code and print a link to BUILDING_FEATHER.md if it failed or something 🙂